argocd_instance/setting-up-argocd-instance.adoc

```diff
@@ -8,17 +8,26 @@ toc::[]
 
 By default, {gitops-title} installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions for managing certain cluster-scoped resources. This default Argo CD instance is also called as the default cluster-scoped instance.
 
+To prevent the default Argo CD instance from starting in the `openshift-gitops` namespace, you can use the `openshift-gitops-operator` subscription and configure the `DISABLE_DEFAULT_ARGOCD_INSTANCE` environment variable in it by setting the value to `true`.
+
```

Comment:
Wondering if we should mention it as `setting the string value to `"true"``

Reply:
Addressing it right away.